src/product-service/handlers/getProductsList.ts

```diff
@@ -0,0 +1,19 @@
+import products from '../data.json'
+export const getProductsList = async () => {
+  try {
+    console.log("products:", products)
+    return {
+      statusCode: 200,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+      },
+      body: JSON.stringify(
+        products
+      ),
+    };
+  } catch (err) {
+    if (err instanceof Error) {
```

Comment:
We should handle any errors as http errors for HTTP lambda handler

The same for other lambda function